tests/activemq5-unit-tests/src/test/java/org/apache/activemq/transport/failover/FailoverPriorityTest.java

```diff
@@ -37,6 +37,7 @@
 import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import java.lang.invoke.MethodHandles;
```

Comment:
This one doesn't seem like it needs the import, or it should actually use it in the logger declaration below.